src/commands/lambda/renderers/flare-renderer.ts

```diff
@@ -13,3 +20,53 @@ export const renderLambdaFlareHeader = (isDryRun: boolean) => {
 
   return `\n${prefix}🐶 Generating Lambda flare to send your configuration to Datadog\n`
 }
+
+/**
+ * @param error an error message
+ * @returns the provided error prefixed by {@link errorTag}.
+ *
+ * ```txt
+ * [Error] The provided error goes here!
+ * ```
+ */
+export const renderError = (error: string) => `\n${errorTag} ${error}\n`
+
+/**
+ * @param warning the message to warn about.
+ * @returns the provided warning prefixed by {@link warningTag}.
+ *
+ * ```txt
+ * [Warning] The provided warning goes here!
+ * ```
+ */
+export const renderWarning = (warning: string) => `${warningTag} ${warning}\n`
```

Comment:
Where is thing being used?
The same goes for the constants below.

Reply:
These are methods I plan on using later in the project. Should they be removed right now?

Reply:
If we are not using them now, let's not add them until we get there.